bridge/contracts/nftbridge/NFTBridge.sol

```diff
@@ -401,7 +292,7 @@ contract NFTBridge is
     address tokenCreator = getTokenCreator(tokenAddress, tokenId);
 
     address payable sender = _msgSender();
-    // Transfer the tokens on IERC20, they should be already Approved for the bridge Address to use them
+    // Transfer the tokens on IERC721, they should be already Approved for the bridge Address to use them
     IERC721(tokenAddress).safeTransferFrom(sender, address(this), tokenId);
```

Comment:
```suggestion
    IERC721(tokenAddress). transferFrom(sender, address(this), tokenId);
```
As we are sending it to our address, we know that we support ERC721 and it is chepear as it does not perform validations